### PR TITLE
tirocinio/plugin-qr-code

### PR DESCRIPTION
**Pull request by Zhin1797**
-Added qr-code plugin to RLN

### CODE REVIEW

File: packages/plugins/qr-code/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@restlessness/plugin-s3",
```
Comment (antoniogiordano) [inline packages/plugins/qr-code/package.json:2]:
```suggestion
  "name": "@restlessness/plugin-qr-code",
```

File: packages/plugins/qr-code/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "version": "0.0.0",
+  "main": "dist/index.js",
+  "author": "Getapper",
+  "license": "MIT",
+  "devDependencies": {
+    "@types/aws-sdk": "2.7.0",
```
Comment (antoniogiordano) [inline packages/plugins/qr-code/package.json:8]:
```suggestion
```

File: packages/plugins/qr-code/src/qr-code/index.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import QRCode from "easyqrcodejs-nodejs";
+
+export const useQrCode = async (options) => {
+    const qrcode = new QRCode(options);
+    return await qrcode.toDataURL();
+};
```
Comment (antoniogiordano) [inline packages/plugins/qr-code/src/qr-code/index.ts:6]:
```suggestion
export const useQrCode = async (options: type???) => {
    return new QRCode(options);
};
```
Let's add some TS types, and let's return all the object, then project can choose how to use it.

File: packages/plugins/qr-code/package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "@types/node": "13.7.1",
+    "@typescript-eslint/parser": "2.25.0",
+    "eslint": "6.8.0",
+    "rimraf": "3.0.2",
+    "typescript": "3.8.3",
+    "yup": "0.29.1"
+  },
+  "peerDependencies": {
+    "aws-sdk": "^2.664.0"
+  },
```
Comment (antoniogiordano) [inline packages/plugins/qr-code/package.json:17]:
```suggestion
```

**Review by antoniogiordano — COMMENTED** [review]
Usually you can find npm packages types by installing npm i @types/easyqrcodejs-nodejs (@types + / + èackage name) did you try?

**Comment by Zhin1797** [comment]
> Usually you can find npm packages types by installing npm i @types/easyqrcodejs-nodejs (@types + / + èackage name) did you try?

I've just tried it and i receive a 404 error, also in their class they use a parameter of type any inside the constructor

**Review by antoniogiordano — APPROVED** [review]
Ok sounds good!

**Pull request merged**